Spanner/src/Database.php

```diff
@@ -859,21 +790,9 @@ public function transaction(array $options = []): Transaction
             throw new \BadMethodCallException('Nested transactions are not supported by this client.');
```

Comment:
Let's import the `BadMethodCallException` instead of using it from the root. I know this is not a change made by you but if we catch them, I think they are easy wins.